packages/baseclient/src/config/config.js

```diff
@@ -1,13 +1,16 @@
 const basePath = window.location.origin +
   window.location.pathname.match(/^\/[\w-]*\/?/)[0];
+const buildPath = window.location.origin +
+  window.location.pathname.match(/^(\/[\w-]*)*\/\/?/)[0]
```

Comment:
```suggestion
  window.location.pathname.match(/^(\/[\w-]*)*\/\/?/)[0];
```

Reply:
We should have a linting rule enforcing one style. If possible create an issue and probably also fix it while you're at it :wink: